viz_config.yml

```diff
@@ -19,5 +19,10 @@ secondary_geoms:
     xlim: [275, 300]
     ylim: [16, 30]
 
+# Change will cause refetch of entire nhdplus for view_polygon
+fetch_simplification_tolerance_m: 500
```

Comment:
I ended up doing this to cut the output file size from the fetcher down. It went from over 300MB to a little over 100MB.